### DISCUSSION TITLE
Accessibility on Mastodon - where does it exist and where would it still need to be expanded?

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#18782] [General]
Accessibility. Many people are very dependent on a major expansion of accessibility, in the analogue as well as in the digital sector. Without accessibility, their participation in analogue and digital projects is either made very difficult or completely impossible (because people are essentially dependent on barrier-free or low barrier offers). So I think it should also have a certain value in digital projects. 

The goal would be to collect points where the Mastodon user interface has already been developed barrier-free or at least with low barriers, and to point out points where there is still a need for improvement, so that this is set as the focus of future versions. 

Thus my two questions: 

- Where is the Mastodon user interface already accessible or at least (very) barrier-free?
- Where should the Mastodon user interface be revised with regard to accessibility?

**Comment by jpdevries** [comment]
Has Mastadon been audited for accessibility yet? My colleagues and I have experience with this and we'd like to help.

**Comment by Sirablopp** [comment]
I would appreciate if the feeds could be adjusted in size, especially in the advanced browser layout, like it is suggested here:

https://github.com/mastodon/mastodon/discussions/19864#discussion-4542753

Additionally, being able to adjust text size would be awesome for a better readability.
